src/components/mosaic-gallery/MosaicGallery.tsx

```diff
@@ -48,11 +48,12 @@ const MosaicGallery: React.FC<IMosaicGalleryProps> = ({onClick}) => {
       }
 
       return {
+        key: image.id,
```

Comment:
why have you added the same value with two different keys?

Reply:
I use id field for later finding respective image, and the key field is used by React during reconciliation.

Reply:
Can't you use the key parameter to find respective image?